crypto/ct/ct_log.c

```diff
@@ -259,6 +259,18 @@ int CTLOG_STORE_load_file(CTLOG_STORE *store, const char *file)
     return ret;
 }
 
+int CTLOG_STORE_add_log(CTLOG_STORE *store, const CTLOG *log)
+{
+    if (!sk_CTLOG_push(store->logs, log)) {
+        goto mem_err;
+    }
+    return 1;
+
+mem_err:
+    ERR_raise(ERR_LIB_CT, ERR_R_MALLOC_FAILURE);
+    return -1;
```

Comment:
I'd like to avoid `goto` in such a short function and return 0, not -1, on error

Reply:
Done.